src/tests/tests/yellowlist.js

```diff
@@ -0,0 +1,112 @@
+/* globals badger:false */
+
+(function() {
+
+  function get_ylist() {
```

Comment:
Let's standardize on "yellowlist", I think it's a better descriptor as this isn't just about cookies.